jitterdebugger.c

```diff
@@ -224,6 +227,17 @@ static void dump_stats(FILE *f, struct stats *s)
 			fprintf(f, "        \"%u\": %lu", j, s[i].hist[j]);
 			comma = 1;
 		}
+		for (j = 0; j < s[i].hist_overflows_last; j++) {
+			if (j+1 != s[i].hist_overflows_last
+			&& s[i].hist_overflows[j] == s[i].hist_overflows[j+1])
+			{
```

Comment:
This looks like you try hard to avoid to hit the 80 chars 'rule' and the result is not so nice. Maybe moving this block and the above into a new function? So you would call something like dump_overflows()?